keyboards/clickety_split/leeloo/keymaps/default_led/rules.mk

```diff
@@ -0,0 +1,3 @@
+# Intentionally left blank.
```

Comment:
This file should be removed.

Reply:
Thank you @waffle87, I'll remove this file as well.